cmake/CodeFormat.cmake

```diff
@@ -0,0 +1,45 @@
+if(NOT CLANG_FORMAT_EXECUTABLE)
+    find_program(CLANG_FORMAT_EXECUTABLE
+        NAMES
+        clang-format-7
+        clang-format-mp-7
```

Comment:
Should we use clang-format-8 instead

Reply:
I could not find it regular repos in Ubuntu.

Reply:
http://apt.llvm.org/

Reply:
Then I don't care. But we would need to rebuild the container as it carries 7.